### PR TITLE
chore: release 7.4.3

### PR DESCRIPTION
**Pull request by yjyoon-dev**
### Code Refactoring

- add corner option for where the ad choices icon is
- **ndarichmedia:** add English resources to Richmedia AD

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I updated the docs if needed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
-## [7.4.2] (2024-03-27)
+
+## 7.4.3 (2024-04-26)
+### Code Refactoring
+- add corner option for where the ad choices icon is
+- **ndarichmedia:** add English resources to Richmedia AD
```
Comment (kwonjae-lee) [inline CHANGELOG.md:6]: nam에는 ndarichmedia 모듈이 없어서 제외되어도 될 것 같습니다

Comment (yjyoon-dev) [reply CHANGELOG.md]: 넵 제외했습니다~!

**Review by kwonjae-lee — APPROVED** [review]
확인 했습니다~

**Review by kidoohan — APPROVED** [review]
확인했습니다~

**Review by euisong-jung-navercorp — APPROVED** [review]
확인했습니다~

**Pull request merged**